internal/services/sast-metadata.go

```diff
@@ -0,0 +1,75 @@
+package services
+
+import (
+	"strings"
+	"sync"
+
+	commonParams "github.com/checkmarx/ast-cli/internal/params"
+	"github.com/checkmarx/ast-cli/internal/wrappers"
+)
+
+const (
+	BatchSize = 200
+)
+
+func GetSastMetadataByIDs(sastMetaDataWrapper wrappers.SastMetadataWrapper, scanIDs []string) (*wrappers.SastMetadataModel, error) {
```

Comment:
check if reference needed and if it better performance

Reply:
Because  SastMetadataModel could be a big object, in this case, it is better to use a pointer.